sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -220,6 +220,24 @@ def display_timezone(self, value):
     """
     self._display_timezone = value
 
+  @property
+  def specified_cache_dir(self):
+    """The cache directory specified by the user.
+
+    Defaults to None.
+    """
+    return self._specified_cache_dir
+
+  @specified_cache_dir.setter
+  def specified_cache_dir(self, value):
+    """Sets the cache directory.
+
+    Defaults to None.
+
+    Example::
+      interactive_beam.options.specified_cache_dir = "gs://"
```

Comment:
Can we add examples for local disk too?

- /Users/username/my/cache/dir

Also, it might improve the readability to make the GCS path example "gs://my-gcs-bucket/cache/dir".